scripts/test

```diff
@@ -15,4 +15,7 @@ if not_installed ginkgo; then
 fi
 
 cd $s3_resource_dir
+
+counterfeiter -o fakes/fake_s3client.go .  S3Client
```

Comment:
Don't think we should add this; usual workflow is `go generate ./...` after changing interfaces.